src/wp-includes/webfonts.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Webfonts API: Webfonts functions
+ *
+ * @since 5.9.0
+ *
+ * @package WordPress
+ * @subpackage Webfonts
+ */
+
+/**
+ * Instantiates the webfonts controller, if not already set, and returns it.
+ *
+ * @since 5.9.0
+ *
+ * @return WP_Webfonts_Controller Instance of the controller.
+ */
+function wp_webfonts() {
+	static $instance;
+
+	if ( ! $instance instanceof WP_Webfonts ) {
+		$instance = new WP_Webfonts_Controller(
+			new WP_Webfonts_Registry(
+				new WP_Webfonts_Schema_Validator()
+			),
+			new WP_Webfonts_Provider_Registry()
+		);
+		$instance->init();
+	}
+
+	return $instance;
+}
+
+/**
+ * Registers a webfont collection.
+ *
+ * @since 5.9.0
+ *
+ * @param array $webfonts Webfonts to be registered.
+ *                        This contains ar array of webfonts to be registered. Each webfont is an array.
+ *                        See {@see WP_Webfonts_Registry::register()} for a list of supported arguments for each webfont.
+ */
+function wp_register_webfonts( array $webfonts ) {
+	// Bail out if the webfonts collection is empty.
+	if ( empty( $webfonts ) ) {
+		return;
+	}
+
+	foreach ( $webfonts as $webfont ) {
+		wp_webfonts()->webfonts()->register( $webfont );
+	}
+}
```

Comment:
Setting a default value removes the need for the subsequent conditional as if `webfonts` is empty the `foreach` loop is never executed.

 
```suggestion
function wp_register_webfonts( array $webfonts = array() ) {
	foreach ( $webfonts as $webfont ) {
		wp_webfonts()->webfonts()->register( $webfont );
	}
}
```

Reply:
The pattern used is consistent with other Core API implementations, such as the `wp_styles()`,  `wp_scripts()`,  and `wp_sitemaps_get_server()`. I think for this API, it should be consistent.